packages/components/src/__actions__/Button/v3/Button.tsx

```diff
@@ -45,11 +50,14 @@ export const Button = forwardRef(
       isPending,
       hasHiddenPendingLabel: propsHasHiddenPendingLabel = false,
       pendingLabel,
+      isReversed,
       ...restProps
     }: ButtonProps,
     ref: React.ForwardedRef<HTMLButtonElement>
   ) => {
-    const isReversed = useReversedColors()
+    const shouldUseReverse = useReversedColors()
+    const isReversedVariant =
+      isReversed !== undefined ? isReversed : shouldUseReverse
```

Comment:
You can do this shorthand (null-coalescing) :)
```suggestion
    const isReversedVariant = isReversed ?? shouldUseReverse
```

Reply:
Oh even better 😉